package.json

```diff
@@ -50,9 +50,9 @@
   },
   "devDependencies": {
     "babel-eslint": "^4.1.5",
-    "react-native-vector-icons": "*",
```

Comment:
It was giving peerDependency issues and bundling `react-native` with npm 2.x so I think that `mock-fs` as a trade-off is still more lightweight ;)